src/styles/blocks/button.scss

```diff
@@ -0,0 +1,32 @@
+.button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  box-sizing: border-box;
+  width: 100%;
+  height: 56px;
+  border: 1px solid $colorRed;
+
+  font-family: Cinzel, serif;
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 27px;
+  text-decoration: none;
+  background-color: $colorRed;
+  color: #fff;
+
+  cursor: pointer;
+  transition: all 0.3s;
```

Comment:
Always add transition property. if transition property is all, it's hard for the browser to control all properties